old-feed.user.js

```diff
@@ -36,23 +40,46 @@
     news.appendChild(followingFeedWrapper);
 
     const picker = document.createElement("div");
-    picker.classList.add("color-border-default");
-    picker.style = "border-bottom: 1px solid; margin-bottom: 1em";
     news.insertBefore(picker, feedContainer);
-    picker.innerHTML = `<button class="Button" data-show="following">Following</button><button class="Button" data-show="forYou">For you</button><span style="float:right">Loading...</span>`;
-    const loadingIndicator = picker.querySelector("span");
+    picker.innerHTML = `
+        <div class="mb-3">
+            <nav class="overflow-hidden UnderlineNav">
+                <ul class="UnderlineNav-body">
```

Comment:
How'd you even find these classes? I poked around for a while before giving up and making my own...

Reply:
It's from the repo code page:

![image](https://github.com/Gerrit0/old-github-feed/assets/24798198/530d4002-f371-44a2-9f67-9ff50593824b)

I just removed all unneeded classes and icons and left the bare minimum to keep the same functionality.